### PR TITLE
Cryptocurrency, remove Etherium as currency for 2655

### PR DESCRIPTION
**Pull request by jkrobles**
Issue: #2655

I removed Etherium as a currency since it is invalid. I added ether instead and tested that any conversion including ehterium is undef. 

https://duck.co/ia/view/cryptocurrency
Maintainer: @claytonspinner

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by tagawa** [comment]
Hi @jkrobles. Thanks for sending this pull request. I saw issue https://github.com/duckduckgo/zeroclickinfo-spice/issues/2655 from @fzzr- and agree that Ether should be added for correctness, but I also think we should keep the existing triggers. Thinking about user behaviour, it's likely that people using the query "1 ethereum in btc" are probably looking for an Ether to BTC conversion, so we should still show them the conversion answer.

Come to think of it, we should probably cover both spellings as well - etherium and ethereum.

File: t/Cryptopcurrency.t [hunk 1]
```diff
@@ -128,7 +128,7 @@ ddg_spice_test(
     # Things that should probably work but it doesn't at the moment.
     'ppc ftc 400' => undef,
     '499 nmc = ? usd' => undef,
-    
+    '500 etherium in btc' => undef,
```
Comment (GuiltyDolphin) [inline t/Cryptopcurrency.t]: We should do as @tagawa said and update this test to be a valid query.

Also add tests for any new triggers you have introduced.

**Comment by jkrobles** [comment]
@tagawa  and @GuiltyDolphin I believe I have added the changes you wanted. I appreciate the help with this issue.

**Comment by tagawa** [comment]
Thank you @jkrobles - looks good to me.

File: share/spice/cryptocurrency/cryptocurrencylist.txt [hunk 2]
```diff
@@ -219,7 +219,7 @@ enrg,energycoin,energy coin,
 eqx,equinoxcoin,equinox coin,
 erm,ermiscoin,ermis coin,
 etc,eternalcoin,eternal coin,
-eth,etherium,etherium coin,
+eth,ether,etherium,etherium coin,ethereum,
```
Comment (GuiltyDolphin) [inline share/spice/cryptocurrency/cryptocurrencylist.txt]: Oh, doh! You were right to remove `ethereum`; checking the original issue and some sources it does seem ethereum isn't a currency. We should probably keep `ethereum coin` though.

Comment (tagawa) [inline share/spice/cryptocurrency/cryptocurrencylist.txt]: People searching for `10 ethereum to bitcoin` - even though the naming is incorrect - would expect to see a conversion for ether, so I'd keep it in to cover such queries.

Comment (GuiltyDolphin) [inline share/spice/cryptocurrency/cryptocurrencylist.txt]: @tagawa True. We should probably update `etherium coin` to `ethereum coin` though (or add another trigger).

Comment (tagawa) [inline share/spice/cryptocurrency/cryptocurrencylist.txt]: My vote is to add another trigger to cover everything.

**Comment by jkrobles** [comment]
The new triggers have been added.

**Comment by tagawa** [comment]
Sorry to make extra work for you! Looks good to me.

**Comment by GuiltyDolphin** [comment]
@claytonspinner You're the maintainer for this - you happy with the changes?

**Comment by fzzr-** [comment]
Imho,
* The `Ethereum` or `Ξthereum` is a network & protocol with built-in currencies named `Ether`, `Wei`,...
* The `Etherium` is a [game](http://etherium-thegame.com) for example, not an Eth...

Imho, we can have the wrong name `Etherium` in order to teach people correct name of the __Ethereum__.

**Comment by fzzr-** [comment]
IMHO, what is on screenshot isn't normal.

screen shot 2016-04-27 at 21 18 03

- - -

But correction "We have included.." with correct name is great 👍 

screen shot 2016-04-27 at 21 23 40

**Comment by claytonspinner** [comment]
@GuiltyDolphin @jkrobles Thanks for making the change! Looks good to me :+1:

**Comment by kewde** [comment]
Hmm,

I've done some previous work on this plugin and I believe something has changed for the worse.

Try this for example;

**Working**
> 20 BTC

**Not working**
> 20 ETH
> 20 eth to btc
> 20 BTC to ETH

I believe I experienced this issue before, it's due to too many input in the cryptocurrencylist.txt for ethereum I believe, 3 max. When we changed names from ShadowCoin to ShadowCash I also tried to add more and it bricked the triggers.

**Pull request merged**